elfpy/utils/sim_utils.py

```diff
@@ -99,10 +98,10 @@ def get_simulator(
 
 
 def get_initialized_market(
-    pricing_model: PricingModel,
-    block_time: BlockTime,
+    pricing_model: hyperdrive_pm.HyperdrivePricingModel | yieldspace_pm.YieldspacePricingModel,
```

Comment:
i feel like we should rename this function to `get_initialized_hyperdrive_market` since it's only for hyperdrive markets it looks like